apps/web/outstatic/content/docs/fetching-data.md

```diff
@@ -18,20 +18,21 @@ To fetch all your collections, without the documents, use the `getCollections` f
 
 ## Fetching documents
 
-To fetch all the documents from a collection, you should use the `getDocuments` function inside your `getStaticProps` or `getServerSideProps` functions. The `getDocuments` function accepts two parameters: the name of the collection as a string, and an array with the fields to be retrieved. It returns an array of Documents sorted by date, with the most recent `publishedAt` documents first.
+To fetch all the documents from a collection, you should use the `getDocuments` function inside your `getStaticProps` or `getServerSideProps` functions. The `getDocuments` function accepts three parameters: the name of the collection as a string, an array with the fields to be retrieved, and a boolean that determines if we fetch drafts. It returns an array of Documents sorted by date, with the most recent `publishedAt` documents first.
 
 ```javascript
 // Fetching all documents from a collection
 
 export const getStaticProps = async () => {
+  const showDraftPgs = process.env.SHOW_DRAFTS;
   const allPosts = getDocuments('posts', [
     'title',
     'publishedAt',
     'slug',
     'coverImage',
     'description',
     'author'
-  ])
+  ],showDraftPgs)
```

Comment:
To avoid having a bunch of follow-on options, it probably makes more sense for this to be an options object.

`getDocuments(category, fields = [], options = {})` with `options.drafts = true` to include drafts. This way if we introduce other things like a page property, we don't go beyond the two optional properties in the signature.

Reply:
I agree with @jakobo. I would go further to suggest that instead of `options.drafts` we should have `options.status` with values such as `published`, `draft`, `all`. In the case someone wants to fetch just drafts too.

Reply:
One more note. I was reading the Netlify CMS docs and they have an [Editorial Workflow](https://www.netlifycms.org/docs/configuration-options/#publish-mode) mode which could be worth exploring for QA or revisions.

Reply:
changed to use the object with a string as the key, it would be nice to have the option to pass configuration initially and be able to specify a status array instead of just draft/published because ive seen some workflows that would need a review step if you are working with multiple ppl, I think that would be a separate pr that can extend this options helper if we want to go that route